### DISCUSSION TITLE
How long it will take to Get Student pack from a New Github Account

### DISCUSSION THREAD

**Discussion by safnimj** [community/community#63998] [New to GitHub] [answered]
I newly created an Account in GitHub when I tried to apply for the Students Developer pack it shows " the account was too recently Created " Can I get any help...

Thank you 😊

**Comment by GhostOf0days** [comment, marked as answer]
I would wait 7 days after the creation of your Github account. It's not really mentioned in requirements to have a dated account. It should work without, but I would just wait those 7 days.
